src/app/components/applications/SSV/ValidatorSuccessScreen/ValidatorSuccessScreen.tsx

```diff
@@ -51,7 +51,7 @@ const ValidatorSuccessScreen = () => {
           <Grid item container className={classes.Wrapper}>
             <Grid item className={classes.Text}>Your new validator is managed by the following cluster:</Grid>
             <Grid container item className={classes.ClusterID}>
-              <Typography>Validator Cluster {longStringShorten(clusterHash, 4)}</Typography>
+              <Typography>Validator Cluster {longStringShorten(clusterHash, 4).replace(/^0x/, '')}</Typography>
```

Comment:
It was not fixed according to new format of shorten function

Reply:
@meshin-blox 
https://github.com/bloxapp/ssv-web/pull/136
please merge it before, I make the pull from stage and fix it

Reply:
done